### DISCUSSION TITLE
How do I change the trash can directory?

### DISCUSSION THREAD

**Discussion by tkm5** [andreafrancia/trash-cli#308] [General]
The default location is ```~/.local/share/Trash/```, but I would like to change it to ```~/.Trash```

**Comment by andreafrancia** [comment]
This is a change that is not supposed to supported because the results will be not usable with other programs that uses the [trash specification](https://specifications.freedesktop.org/trash-spec/trashspec-1.0.html) like Trash Can you will find in Gnome or KDE.

You can use the `--trash-dir` option to change the trash directory, but you need to pass it to each each trash-* command, though you can set aliases in your `.bashrc`/`.zshrc`. 

trash-cli is not designed to be interoperable with the macOS Bin which is "located" in the ~/.Trash directory. 
I'm not interested to make it interoperable because there already an utility that does that macOS does not provide some things I don't want to handle (for example in macOS can record the original location directory but is not able to record correctly the basename of a file when in the trash there is already another trashed file with the same basename).

If you are using macOS and you like your file trashed from the command line to go in the Finder trash you may want try the trash utility by Ali Rantakari (http://hasseg.org/trash), be sure to always use the `-F` option that tell the program to using Finder instead of the system API. Only Finder is able to record the original location of a trashed file. The system API will discard that information.

**Comment by andreafrancia** [comment]
From MacOS 14 the OS provides a "trash" command present in /usr/bin/trash that seems fully integrated with the Bin you see in the Desktop.
